GoogleForCreators/web-stories-wp
# Remove all poster fallback logic

## Summary

As the title says, this PR removes all logic related to using a fallback poster image if none is provided by the user.

## Relevant Technical Choices

This makes all stories without a poster image invalid AMP documents.

## To-do



## User-facing changes



## Testing Instructions

1. Create a new story
1. Publish story
1. View story on frontend
1. See that story is invalid AMP because of a missing poster image.


---



Fixes #3671
